tests/Australia/AustraliaDayTest.php

```diff
@@ -75,22 +98,33 @@ public function testHolidayType(): void
     }
 
     /**
-     * Returns a list of test dates
+     * Returns a list of random test dates used for assertion of the holiday defined in this test
      *
      * @return array list of test dates for the holiday defined in this test
+     * @throws Exception
      */
     public function HolidayDataProvider(): array
+    {
+        return $this->generateRandomDates(1, 26, $this->timezone);
+    }
+
+    /**
+     * Returns a list of test dates
+     *
+     * @return array list of test dates for the holiday defined in this test
+     */
+    public function SubstituteHolidayDataProvider(): array
```

Comment:
The calculateHoliday function is not only used for 'Australia Day'. Ideally this test should also include assertions for other holidays that will get substituted.